### ISSUE TITLE
multiselect list option detach and moves upward when using browser scroll bar.

### ISSUE THREAD

**Issue by momer79** [ehynds/jquery-ui-multiselect-widget#647]
I am using IE and Fire Fox browser, multiselect dropdown is functioning well apart from one following issue;
If i click on drop down list control then it shows list options which is fine , while it is open if i scroll the browser down the option list detach from the drop down control and moves upwards as shown in attached image.
I have tried fixed css method but no luck.

Is this a known issue? and is there any fix available?

![ddl](https://cloud.githubusercontent.com/assets/9468647/16160665/564fd4a2-34c1-11e6-8b64-53bbf6981767.png)

**Comment by mlh758** [comment]
I just tried this in Chrome and didn't have an issue. Let me try IE and Firefox.

**Comment by mlh758** [comment]
Hm, not seeing that happen in IE either. I'm using jQuery 2.2.0, jQuery UI 1.11.4, and the UI smoothness theme: https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css

What versions are you pulling in on your page?

**Comment by momer79** [comment]
I am using MultiSelect UI Widget 1.13 and Jquery version 1 and 2.1.1.

**Comment by mlh758** [comment]
This can also happen in Chrome, I just noticed it happening in a fiddle used in another issue: https://jsfiddle.net/dr7d7h84/9/

**Comment by mlh758** [comment]
@momer79 Okay I think I have an idea for you. This is moving around due to the position utility trying to give you a clean fit on the page. Playing around with the collision options might give you a better fit for the menu.

`$(this).multiselect({position: {my: "center top", at: "center",collision: "none"}});`

Is roughly the default. You can use the options [here](http://api.jqueryui.com/position/)

**Comment by momer79** [comment]
it only happens when i scroll down the page through mouse scroll button. it drags the multi select option box with it

**Comment by mlh758** [comment]
Does this still happen if you upgrade to [multiselect version 1.14](https://github.com/ehynds/jquery-ui-multiselect-widget/releases/tag/1.14)?

**Comment by momer79** [comment]
Yes i have tried with 1.14

**Comment by schittuluri1** [comment]
Hi @momer79  have you been able to fix this issue?
